### PR TITLE
cmd/install: check time of last update if FormulaUnavailableError

### PR DESCRIPTION
**Pull request by dunn**
A second attempt at #39794, addressing the problems with that PR.

This was the least invasive place I could think to put the reminder while still having it be useful.

### CODE REVIEW

File: Library/Homebrew/cmd/install.rb [hunk 1]
```diff
@@ -91,6 +91,14 @@ def install
         puts_columns(search_formulae(query))
         puts "Searching taps..."
         puts_columns(search_taps(query))
+
+        if HOMEBREW_REPOSITORY.exist? &&
+           (Time.now.strftime("%s").to_i - `stat -f "%m" #{HOMEBREW_REPOSITORY}/.git/refs/heads/master`.to_i) > 172800
```
Comment (jacknagel) [inline Library/Homebrew/cmd/install.rb]: `HOMEBREW_REPOSITORY` will always exist (it's not the `.git` directory). Also, you can do the stat call in ruby.

```rb
master = HOMEBREW_REPOSITORY.join(".git", "refs", "heads", "master")
if master.exist? && (Time.now.to_i - File.mtime(master).to_i) > 172800
  # ...
end
```

Comment (dunn) [inline Library/Homebrew/cmd/install.rb]: Oh ok, cool.

Is 48 hours a reasonable time for issuing a reminder?

Comment (mikemcquaid) [inline Library/Homebrew/cmd/install.rb]: Would be good to have the magic number (`172800`) broken down a bit more so it's more obvious how long this is.

**Comment by dunn** [comment]
Any further thoughts on this?

**Comment by mikemcquaid** [comment]
Thanks @dunn!

**Pull request closed without merging**